### DISCUSSION TITLE
Ammonite-repl (in core) installs problematic experimental version that doesn't work

### DISCUSSION THREAD

**Discussion by TJC** [Homebrew/discussions#2117] [Everyday usage] [answered]
Hi,
I'm not sure if this is the right place to report this issue, but I'm taking my best shot. Let me know if this should get reported elsewhere?

Homebrew core contains https://github.com/Homebrew/homebrew-core/blob/master/Formula/ammonite-repl.rb

Fairly recently, this was updated to start using a branch of Ammonite that uses Scala 3 as a base, rather than Scala 2.13.
Unfortunately, there is a significant bug in this version, which totally breaks Ammonite's shell scripting.

See https://github.com/com-lihaoyi/Ammonite/issues/1190

Ideally, Ammonite in Homebrew should be reverted to the stable version, ie. `2.13-2.4.0`, instead of `3.0-2.4.0`

Can that be done?

**Comment by carlocab** [comment, marked as answer]
Sounds fine to me -- just open a pull request. Bump the formula's `revision` to make sure existing users get the updated version of the formula when they `brew upgrade`. (`grep revision` for examples.)

**Reply by TJC** [reply]
Thanks for the quick reply!
I've created https://github.com/Homebrew/homebrew-core/pull/84985 to do this. It looks OK to me, and passes tests, but this is the first time I've made a PR against Homebrew.

**Comment by TJC** [comment]
This has now been fixed upstream in Ammonite, so after the next stable release there, I think the Homebrew recipe could be updated to switch to the scala-3 stream.
